portable/GCC/ARM_CR5/portmacro.h

```diff
@@ -116,18 +116,23 @@
  * handler for whichever peripheral is used to generate the RTOS tick. */
     void FreeRTOS_Tick_Handler( void );
 
-/* If configUSE_TASK_FPU_SUPPORT is set to 1 (or left undefined) then tasks are
-created without an FPU context and must call vPortTaskUsesFPU() to give
-themselves an FPU context before using any FPU instructions. If
-configUSE_TASK_FPU_SUPPORT is set to 2 then all tasks will have an FPU context
-by default. */
-#if( configUSE_TASK_FPU_SUPPORT != 2 )
-    void vPortTaskUsesFPU( void );
-#else
-    /* Each task has an FPU context already, so define this function away to
-    nothing to prevent it being called accidentally. */
-    #define vPortTaskUsesFPU()
-#endif /* configUSE_TASK_FPU_SUPPORT */
+/*
+ * If configUSE_TASK_FPU_SUPPORT is set to 1 then tasks are
+ * created without an FPU context and must call vPortTaskUsesFPU() to give
+ * themselves an FPU context before using any FPU instructions. If
+ * configUSE_TASK_FPU_SUPPORT is set to 2 then all tasks will have an FPU context
+ * by default.
+ */
+    #if ( configUSE_TASK_FPU_SUPPORT == 1 )
+        void vPortTaskUsesFPU( void );
+    #else
+
+/*

```

Comment:
Is the weird alignment result of uncrustify?

Reply:
Yes.